ivy/container/set.py

```diff
@@ -46,21 +81,19 @@ def static_unique_counts(
         )
 
     def unique_counts(
-        self: ivy.Container,
-        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
-        to_apply: bool = True,
-        prune_unapplied: bool = False,
-        map_sequences: bool = False
+            self: ivy.Container,
+            key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+            to_apply: bool = True,
+            prune_unapplied: bool = False,
+            map_sequences: bool = False
     ) -> ivy.Container:
         """
-        ivy.Container instance method variant of ivy.unique_counts. This method 
-        simply wraps the function, and so the docstring for ivy.unique_counts 
+        ivy.Container instance method variant of ivy.unique_counts. This method
+        simply wraps the function, and so the docstring for ivy.unique_counts
         also applies to this method with minimal changes.
-
```

Comment:
This blank line shouldn't be removed, as it fails the numpy-style-docstring test